### ISSUE TITLE
Improve app color scheme

### ISSUE THREAD

**Issue by riesinger** [glidingthroughspace/paper-tracker#142]
The purple on-black color scheme has a contrast ratio of < 1.5. To meet _basic_ design recommendations, it should _at least_ have a ratio of 4.5.

We might in fact go for a light color scheme, our chosen shade of purple has _much_ more contrast on say `rgb(232, 232, 232)` (4.9, which meets the AA recommendation). 

Aside from recommendations, I personally find the purple almost unreadable on the darker shade of grey we use. Also, the colors somehow "hurt" the eye.

@mheidinger what do you think of a light-theme? Otherwise we'd definitively have to make our greys darker and the purple lighter.
